src/Docker/Runner/DataLoader/DataLoader.php

```diff
@@ -147,8 +149,15 @@ public function loadInputData(InputTableStateList $inputTableStateList, InputFil
         $inputTableResult = new InputTableResult();
         $inputTableResult->setInputTableStateList(new InputTableStateList([]));
         $resultInputFilesStateList = new InputFileStateList([]);
+        $readerOptions = new ReaderOptions(
+            !$this->component->allowBranchMapping(),
+            /* preserve is true only for ABS Workspaces which are persistent (shared across runs) (preserve = true).
+                Redshift workspaces are reusable, but still cleaned up before each run (preserve = false). Other
+                workspaces (Snowflake, Local) are ephemeral (thus the preserver flag is irrelevant for them).
+            */
+            $this->getStagingStorageInput() === InputStrategyFactory::WORKSPACE_ABS
```

Comment:
Everything should be the same as in https://github.com/keboola/docker-bundle/pull/630 except this condition

Reply:
logic checks out ✔️